### DISCUSSION TITLE
Web Navigation evolution

### DISCUSSION THREAD

**Discussion by HectorSancho** [Telefonica/mistica-design#1186] [Proposals]
### Goal

Evolucionar la navegación web para que contemple y escale a arquitecturas más complejas.

### Context

El actual componente no escala.

### Impact

Producción

### Discussion Questions

A priori, podría solucionarse en base a lo desarrollado

- Double Navigation Bar
- Secondary Navigation

### Additional considerations

Parece que hay hueco en desarrollo para encajarlo.

### Do you have a visual proposal?

![Captura de pantalla 2023-04-14 a las 15 09 33](https://user-images.githubusercontent.com/56122343/232052741-79cf842f-7e68-4b92-bcfb-18b8f1855493.png)

**Comment by HectorSancho** [comment]
Figma: https://www.figma.com/file/Os5UfsnhLtQ9rnzmtcX8J4/branch/CZTqBJyKdHnfk5ZJaWiddU/%F0%9F%94%B8-Navigation-Bar-Component-Specs?node-id=1206%3A3528&t=wuHqrv8zIofoRMaW-1

**Comment by HectorSancho** [comment]
Esto aplicaría también a las existentes y creo que son mejoras:

- Posibilidad de utilizarla sin Divider para visualmente poder tener continuidad con el Hero
- Permitir el uso del background alternative

**Comment by yceballost** [comment]
se propone como evolución viendo que otros equipos van a necesitar esta evolución y es una solución que funcionará para muchos otros productos https://github.com/Telefonica/mistica-design/issues/1209
